### ISSUE TITLE
Automatic migration of requirements.txt

### ISSUE THREAD

**Issue by iandees** [kennethreitz/pipenv#165]
Would it make sense to have `pipenv install` in a project directory ask about installing the packages listed in a `requirements.txt` (of the `pip install -r requirements.txt` variety) rather than simply creating an empty pipenv project as it seems to do now?

**Comment by kennethreitz** [comment]
I don't think so, you should manually be curating your packages now.

**Comment by iandees** [comment]
What's the recommended path for someone migrating from requirements.txt to pipenv? To manually go through requirements.txt and type `pipenv install x` for each? (or maybe a `cat | for | ...` shell incantation)

**Comment by kennethreitz** [comment]
yes, or edit the pipfile yourself.

**Comment by kennethreitz** [comment]
requirements.txt should be used as a lockfile, not as a pipfile.

**Comment by iandees** [comment]
It might be good to mention pipenv's relationship with a requirements.txt setup. To me they're pretty similar/overlapping at first read through the docs. I respect that you have a strong opinion about how this works: it'd be good to have that more up front so those that aren't as familiar can learn more.

**Comment by FooBarQuaxx** [comment]
@iandees I think you can do a somthing like ``pipenv install && pipenv run pip install -r requirements.txt``.

**Comment by FooBarQuaxx** [comment]
Yeah, but of course this will not add those dependencies to your Pipfile if that's what you need.

**Comment by sschuberth** [comment]
As kindly [pointed out](https://github.com/kennethreitz/pipenv/issues/11#issuecomment-331073367) by @nateprewitt, the feature to [import from requirements.txt](http://docs.pipenv.org/en/latest/advanced.html#importing-from-requirements-txt) has in fact been implemented now.
